exercises/book-store/description.md

```diff
@@ -0,0 +1,49 @@
+To try and encourage more sales of the 5 different books they sell in a popular series, a bookshop has decided to offer 
+discounts of multiple-book purchases. 
+
+One copy of any of the five books costs 8 EUR. 
+
+If, however, you buy two different books, you get a 5% discount on those two books.
+
+If you buy 3 different books, you get a 10% discount. 
+
+If you buy 4 different books, you get a 20% discount.
+
+If you go the whole hog, and buy all 5, you get a huge 25% discount. 
+
+Note that if you buy, say, four books, of which 3 are different titles, you get a 10% discount on the 3 that 
+form part of a set, but the fourth book still costs 8 EUR. 
+
+Your mission is to write a piece of code to calculate the price of any conceivable shopping basket (containing only books from the same series), giving as big a discount as possible.
```

Comment:
@kytrinyx   I am still looking.  Here is a link to a license, looks like boiler plate.  https://github.com/cyber-dojo/start-points-exercises/blob/master/LICENSE.md

I am still looking around.  I don't know if this license really grants permission for use of exercises or just relating to the site as a whole.